src/FreeAge/common/player.cpp

```diff
@@ -94,6 +94,10 @@ void PlayerStats::UnitChange(UnitType unitType, bool death, int d) {
     unitsDied[static_cast<int>(unitType)] += -d;
   }
 
+  if (IsVillager(unitType)) {
+    villagerCount += d;
+  }
+
```

Comment:
Example of adding a new stat... _it could even be a one liner, but I respect your code style :)_

Reply:
Thanks ;)